auth-web/src/components/auth/account-freeze/AccountOverview.vue

```diff
@@ -85,56 +84,53 @@ import { AccessType } from '@/util/constants'
 </template>
 
 <script lang="ts">
-import { Component, Mixins } from 'vue-property-decorator'
-import { mapActions, mapState } from 'pinia'
+import { Member, Organization } from '@/models/Organization'
+import { defineComponent, onMounted, reactive, toRefs } from '@vue/composition-api'
 import CommonUtils from '@/util/common-util'
 import { FailedInvoice } from '@/models/invoice'
-import { Organization } from '@/models/Organization'
-import Steppable from '@/components/auth/common/stepper/Steppable.vue'
 import { useOrgStore } from '@/stores/org'
 
-@Component({
-  computed: {
-    ...mapState(useOrgStore, [
-      'currentOrganization'
-    ])
-  },
-  methods: {
-    ...mapActions(useOrgStore, [
-      'calculateFailedInvoices'
-    ])
-  }
-})
-export default class AccountOverview extends Mixins(Steppable) {
-  private readonly currentOrganization!: Organization
-  private readonly calculateFailedInvoices!: () => FailedInvoice
-  private formatDate = CommonUtils.formatDisplayDate
-  private nsfFee: number = 0
-  private nsfCount: number = 0
-  private totalTransactionAmount: number = 0
-  private totalAmountToPay: number = 0
-  private totalPaidAmount: number = 0
+export default defineComponent({
+  name: 'AccountOverviewView',
+  emits: ['step-forward'],
+  setup (_, { emit }) {
+    const orgStore = useOrgStore()
+    const currentOrganization: Organization = orgStore.currentOrganization
+    const currentMembership: Member = orgStore.currentMembership
+    const calculateFailedInvoices: any = orgStore.calculateFailedInvoices
+    const downloadNSFInvoicesPDF: any = orgStore.downloadNSFInvoicesPDF
+    const formatDate = CommonUtils.formatDisplayDate
+    const suspendedDate = (currentOrganization?.suspendedOn) ? formatDate(new Date(currentOrganization.suspendedOn)) : ''
```

Comment:
You're going to run into date issues with timezones here, suggestion use Moment instead I think it's already included

Reply:
`moment(currentOrganization.suspendedOn)`